cli/cmd/policy_update.go

```diff
@@ -150,3 +185,37 @@ func updatePolicy(cmd *cobra.Command, args []string) error {
 	cli.OutputHuman("The policy %s was updated.\n", updateResponse.Data.PolicyID)
 	return nil
 }
+
+func policyBulkUpdate(args []string) error {
+	var (
+		policyIds []string
+		err       error
+	)
+	// if no policy ids are provided; prompt a list of policy ids
+	if len(args) == 0 {
+		policyIds, err = promptSetPolicyIDs()
+		if err != nil {
+			return err
+		}
+	} else {
+		policyIds = args
+	}
+
+	var bulkPolicies api.BulkUpdatePolicies
+	for _, p := range policyIds {
+		bulkPolicies = append(bulkPolicies, api.BulkUpdatePolicy{
+			PolicyID: p,
+			Severity: policyCmdState.Severity,
+		})
+	}
+
+	response, err := cli.LwApi.V2.Policy.UpdateMany(bulkPolicies)
+	if err != nil {
+		return err
+	}
+
+	cli.Log.Debugw("bulk policy updated response:", response)
```

Comment:
not the right way to use the logger
```suggestion
	cli.Log.Debugw("bulk policy updated", "response", response)
```